styles/base/inputs/_svg-icon-input.scss

```diff
@@ -13,6 +13,7 @@
 
 .sprk-b-InputContainer__icon-container .sprk-b-InputContainer__icon,
 .sprk-b-TextInputIconContainer .sprk-b-InputContainer__icon,
+.sprk-b-TextInputIconContainer .sprk-c-Icon,
```

Comment:
WAsn't this one of the things that was causing an issue? Didn't @RVMendoza need to remove this is favor of what's on line 15?

Reply:
Hmm I added this back to fix a problem with date picker. I'll touch base with RV

Reply:
Update: I talked with RV and ended up rolling this change back. I talked to Steph about making this a separate Issue.